### PR TITLE
Notification microservice

### PR DESCRIPTION
**Pull request by faad555**
- Set up the basic structure of the service
- it will register with our service registry
- notification mode is created
- Added a service layer (left it empty for now, will add methods as needed)

The service is ready to be built upon. 
Next can be,
- Adding Kafka integration
- Implementing actual notification handling

### CODE REVIEW

File: Microservices/notification-service/src/main/resources/application.properties
Comment (Bourkupine) [inline Microservices/notification-service/src/main/resources/application.properties:1]: might want to use .env's for variables

**Review by Bourkupine — COMMENTED** [review]
Left a comment. This needs more functionality but good start.
I'll branch off this and begin integrating kafka

**Review by M-Byte480 — APPROVED** [review]
If you just do ${NOTIFICATION_PORT} as per Euan's Request, you can complete it

**Review by Bourkupine — APPROVED** [review]
Will approve, from now on can you branch off main in the original repo instead of forking the repo.

**Pull request merged**